tests/vm_startup_test.go

```diff
@@ -0,0 +1,162 @@
+package tests
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	v1 "kubevirt.io/api/core/v1"
+
+	networkv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+)
+
+const timeout = 3 * time.Minute
+const pollingInterval = 5 * time.Second
+const testNamespace = "secondary-test"
+const dnsPort = "31111"
+const dnsIP = "127.0.0.1" // Forwarded to the node port - https://github.com/kubevirt/kubevirtci/pull/867 ,https://github.com/kubevirt/kubesecondarydns/pull/23
+const domain = "vm.secondary.io"
+
+var _ = Describe("Virtual Machines Startup", func() {
+
+	BeforeEach(func() {
+		By("Creating test namespace")
+		ns := k8sv1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: testNamespace,
+			},
+		}
+		err := getClient().Create(context.Background(), &ns)
+		Expect(err).ToNot(HaveOccurred(), "couldn't create namespace for the test")
+	})
```

Comment:
Can be done as BeforeAll / BeforeSuite with a defer to cleanup it instead of the AfterEach
On the other hand, it guarantee the leftovers are cleaned if any, but implicit

Reply:
Since we have only one test, and in the future will maybe have few test. I find this approach cleaner and safer.

Reply:
It is an overkill imo, anyhow we can refactor it later, but imo we should